From 462675a5b797afb411de4506425f12ac6ebdf56a Mon Sep 17 00:00:00 2001
From: babak sarashki <babak.sarashki@windriver.com>
Date: Sun, 3 Nov 2019 14:28:29 -0800
Subject: [PATCH 01/20] Various manual pages changes:

remove LIBEXECDIR from slapd.8
remove references to non-existing manpages (bz 624616)
---
 doc/man/man1/ldapmodify.1 | 3 +--
 doc/man/man5/ldap.conf.5  | 1 +
 doc/man/man8/slapd.8      | 8 ++++----
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/doc/man/man1/ldapmodify.1 b/doc/man/man1/ldapmodify.1
index 19263eb..46a8593 100644
--- a/doc/man/man1/ldapmodify.1
+++ b/doc/man/man1/ldapmodify.1
@@ -397,8 +397,7 @@ exit status and a diagnostic message being written to standard error.
 .BR ldap_add_ext (3),
 .BR ldap_delete_ext (3),
 .BR ldap_modify_ext (3),
-.BR ldap_modrdn_ext (3),
-.BR ldif (5).
+.BR ldif (5)
 .SH AUTHOR
 The OpenLDAP Project <http://www.openldap.org/>
 .SH ACKNOWLEDGEMENTS
diff --git a/doc/man/man5/ldap.conf.5 b/doc/man/man5/ldap.conf.5
index 0c8ec8a..49f1830 100644
--- a/doc/man/man5/ldap.conf.5
+++ b/doc/man/man5/ldap.conf.5
@@ -318,6 +318,7 @@ certificates in separate individual files. The
 .B TLS_CACERT
 is always used before
 .B TLS_CACERTDIR.
+The specified directory must be managed with the OpenSSL c_rehash utility.
 This parameter is ignored with GnuTLS.
 
 When using Mozilla NSS, <path> may contain a Mozilla NSS cert/key
diff --git a/doc/man/man8/slapd.8 b/doc/man/man8/slapd.8
index b554305..22fff11 100644
--- a/doc/man/man8/slapd.8
+++ b/doc/man/man8/slapd.8
@@ -5,7 +5,7 @@
 .SH NAME
 slapd \- Stand-alone LDAP Daemon
 .SH SYNOPSIS
-.B LIBEXECDIR/slapd 
+.B slapd
 [\c
 .BR \-4 | \-6 ]
 [\c
@@ -317,7 +317,7 @@ the LDAP databases defined in the default config file, just type:
 .LP
 .nf
 .ft tt
-	LIBEXECDIR/slapd
+	slapd
 .ft
 .fi
 .LP
@@ -328,7 +328,7 @@ on voluminous debugging which will be printed on standard error, type:
 .LP
 .nf
 .ft tt
-	LIBEXECDIR/slapd \-f /var/tmp/slapd.conf \-d 255
+	slapd -f /var/tmp/slapd.conf -d 255
 .ft
 .fi
 .LP
@@ -336,7 +336,7 @@ To test whether the configuration file is correct or not, type:
 .LP
 .nf
 .ft tt
-	LIBEXECDIR/slapd \-Tt
+	slapd -Tt
 .ft
 .fi
 .LP
-- 
2.17.1

